src/basic_recipes/barplot.jl

```diff
@@ -260,6 +260,6 @@ function Makie.plot!(p::BarPlot)
         highclip = p.highclip, lowclip = p.lowclip, nan_color = p.nan_color,
     )
     if !isnothing(p.bar_labels[])
-        text!(p, labels; align=label_aligns, offset=label_offsets, color=label_colors, font=p.label_font, fontsize=p.label_size, rotation=p.label_rotation)
```

Comment:
to be reverted before merging (`align` needs to be implemented for plotting `LaTeXString`s)